### DISCUSSION TITLE
Photo review

### DISCUSSION THREAD

**Discussion by tjain90** [inspireui/support#5353] [Working on Features]
Hi team, it would be great if photo review support is added. For every ecommerce photo review is must. 
You can add support of 
Woocomere review pro or woocomerce photo review plugin. 

This is essential feature for any successful ecom app.

**Comment by shubhamrane48** [comment]
Yes, This feature is must needed.

**Reply by ecomdeveloperbd** [reply]
Yes, This feature is must needed
